exercises/04_rmarkdown/template.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+output: pdf_document
+params:
+  country_selected: KR  # options: UK, ME, KR 
+  show: false
+header-includes:
+- \usepackage{sectsty} 
+---
+
+```{r logo, echo=FALSE, fig.align="center", out.width="50%"}
+knitr::include_graphics("OECD_logo.jpg")
+```
+
+\allsectionsfont{\centering}
+# Country note: `r params$country_selected`^[Disclaimer: <https://oecdcode.org/disclaimers/territories.html>] 
+
+\newpage
+
+```{r setup, echo=FALSE, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r libraries, echo=FALSE, message=FALSE}
+library(tidyverse)
+library(readxl)
+```
+
+```{r global, echo=FALSE, message=FALSE}
```

Comment:
I would avoid the name `global` because it means something different in Shiny/Stata. `theme` is a good alternative.